deploy/ray-operator/controllers/common/constant.go

```diff
@@ -0,0 +1,27 @@
+package common
+
+const (
+	// use as pod type decision
```

Comment:
decision for what?

Reply:
- For example , decide to create service or not. For now only head will create service.
- Also , it can be used as selector to select pod to do upgrade or roll out